### PR TITLE
Newplant

### PR DESCRIPTION
**Pull request by bomentum**
i just commented out your code Jarod where i made changes

### CODE REVIEW

**Review by jkafkadev — APPROVED** [review]

File: src/pages/index.js [hunk 1]
```diff
@@ -1,198 +1,57 @@
...
-        </div>
-        
-      </main>
-    </>
-  );
+    return (
+        <>
+            <Head>
+                <title>Plant app idk</title>
+                <meta name='description' content='Generated by create next app' />
```
Comment (Jrendl) [inline src/pages/index.js]: we can probably delete this line

File: package.json [hunk 2]
```diff
@@ -16,6 +16,7 @@
     "codehooks-js": "^1.0.10",
     "eslint": "8.37.0",
     "eslint-config-next": "13.2.4",
+    "jwt-decode": "^3.1.2",
```
Comment (Jrendl) [inline package.json:19]: where is this used

Comment (jkafkadev) [reply package.json:19]: Used in the backend/index.js for auth

File: src/pages/index.js (hunk 1 above)
Comment (jkafkadev) [reply src/pages/index.js]: done

**Review by Jrendl — APPROVED** [review]

**Pull request merged**